RadixWallet/Features/AssetTransferFeature/Components/TransferAccountList/ReceivingAccount/Asset/ResourceAsset+View.swift

```diff
@@ -106,3 +110,17 @@ private extension View {
 		}
 	}
 }
+
+private extension ResourceAsset.State.DepositStatus {
+	// FIXME: Localisation
```

Comment:
Waiting on https://github.com/radixdlt/apps-localization-src/pull/445 approval

Reply:
Next time put "FIXME: Strings", that's what we normally use, then it's easy to search in case you forget to update it...